DatadogSessionReplay/Sources/SessionReplay.swift

```diff
@@ -69,6 +69,14 @@ public enum SessionReplay {
                 description: "Datadog SDK must be initialized before calling `SessionReplay.enable(with:)`."
             )
         }
+
+        guard !CoreRegistry.isFeatureEnabled(feature: SessionReplayFeature.self) else {
+            core.telemetry.debug("Session Replay has already been enabled")
```

Comment:
question/ What is the rationale of this telemetry? It reports user error (wrong use of the SDK), hence is not actionable on our side and doesn't say anything about the SDK functioning. In other words: what could we do if seeing these logs 🙂?

Reply:
I believe the intent behind this telemetry is to understand if customers are trying to enable Session Replay multiple times, which could indicate interest in having support for multiple cores with this feature.

@maciejburda: was there any other reasoning behind this telemetry?